packages/components/src/components/hds/reveal/toggle/button.hbs

```diff
@@ -9,5 +9,6 @@
   @icon="chevron-down"
   aria-expanded={{if @isOpen "true" "false"}}
   class={{this.classNames}}
+  aria-describedby={{if @ariaDescribedBy @ariaDescribedBy null}}
```

Comment:
This can be removed, because you assign it as HTML attribute in the invocation in `reveal/index.hbs`
```suggestion
```

Reply:
If you're unsure, try to replace it with this instead `aria-describedby={{if @ariaDescribedBy "hello!" null}}` and you will see that 'hello' is not rendered (how could it, the `@ariaDescribedBy` is not passed to the `<Hds::Reveal::Toggle::Button>` instance)